RxAlamofire/RxAlamofireTests/RxAlamofireTests.swift

```diff
@@ -69,4 +71,40 @@ class RxAlamofireSpec: XCTestCase {
             XCTFail("\(error)")
         }
 	}
+
+    func testProgress() {
+        do {
+            let dataRequest = try request(HTTPMethod.get, "http://myjsondata.com").toBlocking().first()!
+            let progressObservable = dataRequest.rx.progress().observeOn(MainScheduler.instance).replayAll()
```

Comment:
Need to buffer elements since the `progressHandler` won't be set unless the `Observable` is started, and we can't start it using `.toBlocking().toArray()`, or we won't have the chance to call `progressHandler.closure`.